chef/data_bags/crowbar/bc-template-ceph.json

```diff
@@ -6,7 +6,9 @@
       "master": false,
       "disk_mode": "first",
       "config": {
-        "fsid" : ""
+        "fsid" : "",
+        "osds_in_total": 0,
```

Comment:
TBH, I'm confused as to why this one needs to be an attribute. Can't it just be calculated in the recipe?